### DISCUSSION TITLE
WARNING : in directory nforce in OUTCAR is not equal to settings i n INCAR

### DISCUSSION THREAD

**Discussion by Saberve** [deepmodeling/dpgen#1057] [General]
![image](https://user-images.githubusercontent.com/53855907/204586160-4275f942-eccc-4606-9633-63fc53ea7eaf.png)
I'm not sure what this report means.
I should ignore it or adjust INCAR？

**Comment by HuangJiameng** [comment]
It is asserted that `nforce` in a valid OUTCAR should be equal to  `md_nstep` set in INCAR, and this WARNING means the OUTCAR in `sys-0032` is invalid. You may check if your vasp MD simulation is correctly done.

**Comment by biglinn** [comment]
However, I'm not sure what > nforce in OUTCAR refers to. I dont find this term in OUTCAR.
I follow the tourial to keep > md_nsteps in param.json and NSW in INCAR (for md) in agreement.
Can you explain this for more details? Thanks!

**Reply by Saberve** [reply]
I have seen the discussion you submitted, maybe your VASP is not running correctly. You can check the OUTCAR under the folder.

**Reply by biglinn** [reply]
Will it be this reason? I shall check OUTCAR.
And if the probelm lies in VASP erro, what leads to it?  Could be parameters in param.json?

**Reply by Saberve** [reply]
I think this warning is independent of the parameters. It may be because Vasp is unable to calculate some structure, resulting in an unequal `nforce` and `md_nstep`

**Reply by biglinn** [reply]
I got it. I'll check it. Thanks!

**Comment by Normadp** [comment]
Hello, I would like to ask how you solved it in the end？
